lib/logger/err-serializer.ts

```diff
@@ -1,6 +1,7 @@
 import is from '@sindresorhus/is';
 
-export default function errSerializer(err) {
+// TODO: remove any type
+export default function(err: any): any {
```

Comment:
```suggestion
export default function errSerializer(err: unknown): unknown{
```
Same here? Can we change type `any` to `unknown`?

Reply:
Can do, but there is more work required in the function body. I'll reject this suggestion, and make this change locally instead.

Reply:
@ViceIce Okay, so I gave it a go replacing `any` with `unknown` here. However, the logic ended up being rather too complex and if it's okay with you I'd like to leave it for now?

In general I think we should move as much `any` to `unknown` as possible, but I'd like to start with a simpler case